code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -1,14 +1,103 @@
+import Radium from 'radium';
 import React from 'react';
 import { STAGE_PROGRESS_TYPE } from './types';
 import { saveAnswersAndNavigate } from '../../levels/saveAnswers';
+import color from '../../color';
+
+var style = {
+  overviewContainer: {
+    display: 'table-cell',
+    verticalAlign: 'middle',
+    paddingRight: '10px'
+  },
+  headerContainer: {
+    padding: '5px 8px',
+    backgroundColor: color.lightest_gray,
+    border: `1px solid ${color.lighter_gray}`,
+    borderRadius: '5px'
+  },
+  dot: {
+    puzzle: {
+      display: 'inline-block',
+      width: '24px',
```

Comment:
Good catch on `lineHeight`, I originally made this mistake before I saw your comment:

![lineheight_bug](https://cloud.githubusercontent.com/assets/413693/15840493/fc36a6b4-2bfe-11e6-8ec9-77e7a349a80e.png)

It makes me a little nervous that this mistake is so easy to make.  I almost prefer always including units to avoid ambiguity, but I see the benefits of being able to do math with the shorthand.